### ISSUE TITLE
drugInteractions and OncogenicPathways functions not working

### ISSUE THREAD

**Issue by D0g4** [PoisonAlien/maftools#212]
I am not able to run those functions. I saw there was a similar thread for "OncogenicPathways", so I checked the maftools version I was running (1.8.10).
Here is what I am getting:
`> druggable = drugInteractions(maf = OV_1, fontSize = 0.75)
Error in drugInteractions(maf = laml, fontSize = 0.75) : `
`  could not find function "drugInteractions"`

And:
`> OncogenicPathways(maf = laml)
Error in OncogenicPathways(maf = laml) :`
`could not find function "OncogenicPathways"`

Any help on what the issue might be?

`> sessionInfo()`
`R version 3.4.1 (2017-06-30)`
`Platform: i386-w64-mingw32/i386 (32-bit)`
`Running under: Windows 7 x64 (build 7601) Service Pack 1`
`Matrix products: default`
`locale:
[1] LC_COLLATE=Danish_Denmark.1252  LC_CTYPE=Danish_Denmark.1252    LC_MONETARY=Danish_Denmark.1252
[4] LC_NUMERIC=C                    LC_TIME=Danish_Denmark.1252`
`    attached base packages:
[1] grid      parallel  stats     graphics  grDevices utils     datasets  methods   base`
`     
other attached packages:
[1] ComplexHeatmap_1.17.1 BiocInstaller_1.28.0  usethis_1.4.0         devtools_2.0.1        limma_3.34.9         
[6] maftools_1.8.10       Biobase_2.38.0        BiocGenerics_0.24.0`

**Comment by PoisonAlien** [comment]
Hi,
Do you still have this issue ? This should work for versions >1.8

**Comment by D0g4** [comment]
Hi, Poison
Thank you for the feedback!
Yes, I still have the issue. And as you can see from the sessionInfo(), I'm using version 1.8.10.
Would you have any idea on how to solve it?
Thanks in advance.

**Comment by PoisonAlien** [comment]
Can you post your complete `sessionInfo` ?

**Comment by D0g4** [comment]
Sure:

> sessionInfo()
R version 3.4.1 (2017-06-30)
Platform: i386-w64-mingw32/i386 (32-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

Matrix products: default

locale:
[1] LC_COLLATE=Danish_Denmark.1252  LC_CTYPE=Danish_Denmark.1252    LC_MONETARY=Danish_Denmark.1252
[4] LC_NUMERIC=C                    LC_TIME=Danish_Denmark.1252    

attached base packages:
[1] parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] maftools_1.8.10      BiocInstaller_1.28.0 limma_3.34.9         Biobase_2.38.0       BiocGenerics_0.24.0 

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.0                 bindr_0.1.1                XVector_0.18.0             magrittr_1.5              
 [5] cluster_2.0.6              GenomicAlignments_1.14.2   GenomicRanges_1.30.3       IRanges_2.12.0            
 [9] splines_3.4.1              zlibbioc_1.24.0            wordcloud_2.6              mclust_5.4.2              
[13] cowplot_0.9.3              tidyselect_0.2.5           BSgenome_1.46.0            xtable_1.8-3              
[17] colorspace_1.3-2           lattice_0.20-35            gridBase_0.4-7             R6_2.3.0                  
[21] rlang_0.3.0.1              foreach_1.4.4              bibtex_0.4.2               withr_2.1.2               
[25] matrixStats_0.54.0         iterators_1.0.10           assertthat_0.2.0           digest_0.6.18             
[29] tibble_1.4.2               Matrix_1.2-10              ggrepel_0.8.0              RCurl_1.95-4.11           
[33] ComplexHeatmap_1.17.1      compiler_3.4.1             pillar_1.3.1               Rsamtools_1.30.0          
[37] Biostrings_2.46.0          scales_1.0.0               stats4_3.4.1               GetoptLong_0.1.7          
[41] pkgconfig_2.0.2            rstudioapi_0.8             BiocParallel_1.12.0        munsell_0.5.0             
[45] doParallel_1.0.14          rjson_0.2.20               pkgmaker_0.27              GenomeInfoDb_1.14.0       
[49] plyr_1.8.4                 dplyr_0.7.8                stringr_1.3.1              tools_3.4.1               
[53] rngtools_1.3.1             SummarizedExperiment_1.8.1 grid_3.4.1                 circlize_0.4.5            
[57] data.table_1.11.8          gtable_0.2.0               registry_0.5               lazyeval_0.2.1            
[61] survival_2.41-3            crayon_1.3.4               bindrcpp_0.2.2             gridExtra_2.3             
[65] GenomeInfoDbData_1.0.0     rtracklayer_1.38.3         purrr_0.2.5                ggplot2_3.1.0             
[69] reshape2_1.4.3             RColorBrewer_1.1-2         bitops_1.0-6               codetools_0.2-15          
[73] S4Vectors_0.16.0           GlobalOptions_0.1.0        glue_1.3.0                 shape_1.4.4               
[77] NMF_0.21.0                 DelayedArray_0.4.1         stringi_1.2.4              XML_3.98-1.16

**Comment by zx8754** [comment]
Same here:

Code from the manuals:

```R
library("maftools")
laml.maf <- system.file("extdata", "tcga_laml.maf.gz", package = "maftools")
laml <- read.maf(maf = laml.maf)
drugInteractions(maf = laml)

# Error in drugInteractions(maf = laml) : 
#  could not find function "drugInteractions"
```

**sessionInfo()**

```R
R version 3.4.2 (2017-09-28)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252  LC_CTYPE=English_United Kingdom.1252   
[3] LC_MONETARY=English_United Kingdom.1252 LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] maftools_1.4.28      Biobase_2.38.0       BiocGenerics_0.24.0  BiocInstaller_1.28.0

loaded via a namespace (and not attached):
 [1] bitops_1.0-6               matrixStats_0.54.0         bit64_0.9-7               
 [4] doParallel_1.0.14          RColorBrewer_1.1-2         progress_1.2.0            
 [7] httr_1.3.1                 GenomeInfoDb_1.14.0        tools_3.4.2               
[10] R6_2.3.0                   DBI_1.0.0                  lazyeval_0.2.1            
[13] colorspace_1.3-2           GetoptLong_0.1.7           withr_2.1.2               
[16] gridExtra_2.3              tidyselect_0.2.5           prettyunits_1.0.2         
[19] RMySQL_0.10.15             bit_1.1-14                 compiler_3.4.2            
[22] DelayedArray_0.4.1         pkgmaker_0.27              rtracklayer_1.38.3        
[25] scales_1.0.0               NMF_0.21.0                 stringr_1.3.1             
[28] digest_0.6.18              Rsamtools_1.30.0           XVector_0.18.0            
[31] pkgconfig_2.0.2            changepoint_2.2.2          bibtex_0.4.2              
[34] BSgenome_1.46.0            rlang_0.2.2                GlobalOptions_0.1.0       
[37] rstudioapi_0.8             RSQLite_2.1.1              shape_1.4.4               
[40] bindr_0.1.1                zoo_1.8-4                  mclust_5.4.2              
[43] BiocParallel_1.12.0        dplyr_0.7.7                VariantAnnotation_1.24.5  
[46] RCurl_1.95-4.11            magrittr_1.5               GenomeInfoDbData_1.0.0    
[49] wordcloud_2.6              Matrix_1.2-11              Rcpp_0.12.19              
[52] munsell_0.5.0              S4Vectors_0.16.0           stringi_1.1.7             
[55] yaml_2.2.0                 SummarizedExperiment_1.8.1 zlibbioc_1.24.0           
[58] plyr_1.8.4                 grid_3.4.2                 blob_1.1.1                
[61] ggrepel_0.8.0              crayon_1.3.4               lattice_0.20-35           
[64] cowplot_0.9.3              Biostrings_2.46.0          splines_3.4.2             
[67] GenomicFeatures_1.30.3     circlize_0.4.5             hms_0.4.2                 
[70] ComplexHeatmap_1.17.1      pillar_1.3.0               GenomicRanges_1.30.3      
[73] rjson_0.2.20               rngtools_1.3.1             reshape2_1.4.3            
[76] codetools_0.2-15           biomaRt_2.34.2             stats4_3.4.2              
[79] XML_3.98-1.16              glue_1.3.0                 data.table_1.11.8         
[82] foreach_1.4.4              gtable_0.2.0               purrr_0.2.5               
[85] assertthat_0.2.0           ggplot2_3.0.0              gridBase_0.4-7            
[88] xtable_1.8-3               survival_2.41-3            tibble_1.4.2              
[91] iterators_1.0.10           GenomicAlignments_1.14.2   AnnotationDbi_1.40.0      
[94] registry_0.5               memoise_1.1.0              IRanges_2.12.0            
[97] bindrcpp_0.2.2             cluster_2.0.7-1  
```

**Comment by PoisonAlien** [comment]
Okay this is really weird. Could you try re-installing from Github and see if it helps ?

@zx8754 you really have older version of maftools. These functions are only implemented recently in 1.8 onwards. Could you please update the package and let me know ?

**Comment by D0g4** [comment]
That is very weird indeed! I did retrieve it from GitHub last Friday, with no success.
However, I just did it again and is now working fine. :|
Thank you for the troubleshooting, @PoisonAlien!
